addOns/commonlib/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Changed
 - Dependency updates.
+- Maintenance changes.
```

Comment:
This actually affects the alerts.

Reply:
Okay will be more specific

Reply:
Note, I did search and didn't find it in use within the extensions repo.